.github/workflows/Tests.yml

```diff
@@ -694,7 +600,7 @@ jobs:
 
   TOX:
     if: ${{ success() }}
-    needs: [MATS, COVERAGE-MATS, STYLE, COVERAGE, INTEGRATION, DOCS]
+    needs: [MATS, STYLE, COVERAGE, INTEGRATION, DOCS]
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to bump versions too.